src/main/java/com/exadel/discountwebapp/location/vo/city/CityResponseVO.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.discountwebapp.location.vo.city;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CityResponseVO {
+    @NotBlank
+    @Size(min = 2, max = 50)
```

Comment:
There is no need to have validation annotations in response VOs

They won't be used anywhere

Reply:
Removed